### DISCUSSION TITLE
PHP only installs the development php.ini

### DISCUSSION THREAD

**Discussion by shivammathur** [Homebrew/discussions#2719] [Everyday usage] [answered]
PHP formulae currently use the development php.ini. This is fine because mostly homebrew is used for development environments, but it should also install the production one for easy switching if required. 

Currently, the production INI file has to be fetched manually from the source repo, but if it is installed alongside the default development one it would make it easier to switch.

Will be happy to create the PR if adding this is acceptable.

**Comment by SMillerDev** [comment, marked as answer]
Please make a pull request, that way it is much easier to see and discuss the implications

**Reply by shivammathur** [reply]
Done https://github.com/Homebrew/homebrew-core/pull/92489
